org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/serializer/impl/ChangeSerializer.java

```diff
@@ -67,9 +67,11 @@
 		modifications.add(Tuples.create(context, modification));
 	}
 
+	/** some additional effort required in 'endRecordChanges(IAcceptor, SubMonitor)' */
+	static final int additionalEffortInEndRecordChanges = 5; 
```

Comment:
nitpic: should this be a private field?

Reply:
You got me. Worked too much in Xtend in the past.